index.html

```diff
@@ -1540,6 +1553,43 @@ <h2>Event summary</h2>
       </tbody>
     </table>
   </section>
+  <section>
+    <h2>Timestamp behavior</h2>
+    <h2>RTCRtpSender video track frame timestamp effects on outgoing packet NTP and RTP timestamps</h2>
+    <p>
+      The user agent defines a <dfn class="export">frame timestamp</dfn> being the same as the [=capture timestamp=]
+      of the frame being produced on the {{RTCRtpSender}} {{MediaStreamTrack}} if it is set.
```

Comment:
Sorry, I'm not following what this timestamp is by the name "frame timestamp". Not all track sources will insert capture timestamps in the tracks they produce. Is this what you mean by "if it is set"?